test/tests/functional/pbs_offline_vnodes.py

```diff
@@ -104,6 +106,43 @@ def create_multi_vnodes(self, num_moms, num_vnode=3):
             self.server.expect(NODE, {ATTR_NODE_state: 'free'},
                                id=self.moms.values()[i].shortname)
 
+    def verify_other_vnodes_state(self, expected_state):

```

Comment:
Can you please add a doc string for this function. Also, please change the name of the function. Currently it sounds like it is trying to verify some other vnode's state but the function verifies that all the reported vnodes are in expected_state.

Reply:
How about "verify_vnodes_state"?  I will add a doc string.